analysis_options.yaml

```diff
@@ -72,7 +72,6 @@ linter:
     - hash_and_equals
     - implementation_imports
     - iterable_contains_unrelated_type
-    - invariant_booleans
```

Comment:
I saw this was deprecated
https://github.com/dart-lang/linter/issues/811#issuecomment-1289550703

Reply:
@diarmaidlindsay 
That is why I am deleting it. What do you mean?

Reply:
I am supporting the deletion by linking to the official documentation 👍